benchmarks/dynamo/common.py

```diff
@@ -3272,13 +3272,9 @@ def warmup(fn, model, example_inputs, mode, niters=10):
             )
 
             if self.args.export_aot_inductor:
-                t_0 = time.perf_counter()
                 optimized_model_iter_fn = optimize_ctx
```

Comment:
We're just timing how long it takes to assign `optimized_model_iter_fn`, which is pointless, it's always going to be fast.